tests/scalers/cassandra.test.ts

```diff
@@ -12,57 +12,90 @@ const nginxDeploymentName = 'nginx-deployment'
 
 test.before(t => {
     // install cassandra
+    console.log("Install cassandra")
```

Comment:
I know that in one or two scalers we have some console.log, but, do they make sense? 🤔

Reply:
After all those fails, I thought it might be good to put some logs to detect where exactly the test stucks. I can remove them if you wish

Reply:
No no, I don't have any strong opinion about this, I was just sharing my thoughts :)